pages/learn/getting-started.md

```diff
@@ -182,6 +182,7 @@ Your application will then be downloaded and executed by all the devices you hav
 - Manage your device fleet with the use of [configuration][configuration], [environment][service], and [service variables][service].
 - Explore these [example projects][projects] to give you an idea of more things you can do with {{ $names.company.lower }}.
 - If you find yourself stuck or confused, help is just a [click away][help].
+- For an easier way to find and publish apps and projects for edge devices, head to [balenaHub](https://hub.balena.io)
```

Comment:
@somombo Can you instead remove line 183 altogether and add this instead. 

Read more about these [example projects][projects] to give you an idea of more things you can do with {{ $names.company.lower }}

For Line 185, 

```suggestion
- To easily find, deploy and publish your applications for balena supported devices, head to [balenaHub](https://hub.balena.io)
```

Move line 185 a bit higher than other points, as Hub is our most maintained collection of projects to deploy and work with.